src/main/target/common.h

```diff
@@ -202,4 +202,11 @@
 #elif !defined(STM32F7)
 #define MAX_MIXER_PROFILE_COUNT 1
 #endif
+
+#if (MCU_FLASH_SIZE <= 512)
+    #define SKIP_CLI_COMMAND_HELP
+    #undef USE_SERIALRX_SPEKTRUM
```

Comment:
Probably cleaner to move the defines to inside the MCU_FLASH_SIZE > 512 above, instead of using #undef

Reply:
i seen that but elif makes it very hard to read. Probably will clean the whole section

Reply:
Maybe move the elif check to common_post and check for f411, instead of not f7